src/contracts/core/DelegationManager.sol

```diff
@@ -613,40 +668,65 @@ contract DelegationManager is Initializable, OwnableUpgradeable, Pausable, Deleg
                     "DelegationManager._completeQueuedWithdrawal: withdrawalDelayBlocks period has not yet passed for this strategy"
                 );
 
+                // uint256 scalingFactorAtEndOfSlashability = slasher.shareScalingFactorAtEpoch(
+                //     withdrawal.delegatedTo,
+                //     withdrawal.strategies[i],
+                //     epochForEndOfSlashability
+                // );
+                // uint256 shares = SlashingAccountingUtils.scaleDown(withdrawal.shares[i], scalingFactorAtEndOfSlashability);
+                uint256 shares = SlashingAccountingUtils.scaleDown(
+                    withdrawal.shares[i], 
+                    slasher.shareScalingFactorAtEpoch(
+                        withdrawal.delegatedTo,
+                        withdrawal.strategies[i],
+                        epochForEndOfSlashability
+                    )
+                );
+
                 /** When awarding podOwnerShares in EigenPodManager, we need to be sure to only give them back to the original podOwner.
                  * Other strategy shares can + will be awarded to the withdrawer.
                  */
                 if (withdrawal.strategies[i] == beaconChainETHStrategy) {
-                    address staker = withdrawal.staker;
+                    // address staker = withdrawal.staker;
+                    address podOwnerOperator = delegatedTo[withdrawal.staker];
+                    // uint256 scalingFactor = slasher.shareScalingFactor(podOwnerOperator, beaconChainETHStrategy);
+                    // uint256 nonNormalizedShares = SlashingAccountingUtils.scaleUp(shares, scalingFactor);
+                    // TODO: unfortunately there doesn't seem to be a good way to avoid scaling down then up
+                    uint256 nonNormalizedShares = SlashingAccountingUtils.scaleUp(
+                        shares,
+                        slasher.shareScalingFactor(podOwnerOperator, beaconChainETHStrategy)
+                    );
                     /**
                     * Update shares amount depending upon the returned value.
                     * The return value will be lower than the input value in the case where the staker has an existing share deficit
                     */
                     uint256 increaseInDelegateableShares = eigenPodManager.addShares({
-                        podOwner: staker,
-                        shares: withdrawal.shares[i]
+                        podOwner: withdrawal.staker,
+                        shares: nonNormalizedShares
```

Comment:
param name should be `nonNormalizedShares`?

Reply:
yes